stix_shifter_modules/nozomi_vantage/configuration/config.json

```diff
@@ -0,0 +1,37 @@
+{
+  "connection": {
```

Comment:
Can you add a page_size parameter here that defaults to 1000 (or whatever the default is for Nozomi)?

Reply:
Updated as per the suggestion.